draft-ietf-quic-tls.md

```diff
@@ -1110,7 +1112,7 @@ sample.
 To ensure that sufficient data is available for sampling, packets are padded so
 that the combined lengths of the encoded packet number and protected payload is
 at least 4 bytes longer than the sample required for header protection.  The
-ciphersuites defined in {{?TLS13}} - other than TLS_AES_128_CCM_8_SHA256, for
+cipher suites defined in {{!TLS13}} - other than TLS_AES_128_CCM_8_SHA256, for
```

Comment:
I tend to use ? where the reference is normative.

If you follow this discipline and remove the last normative reference, the right thing happens.  But I can't see that happening here, so this change is fine.